services/app-web/src/components/DataDetail/DataDetailContactField/DataDetailContactField.tsx

```diff
@@ -0,0 +1,47 @@
+import { Link } from '@trussworks/react-uswds'
+import {
+    ActuaryContact,
+    StateContact,
+} from '../../../common-code/healthPlanFormDataType'
+import { getActuaryFirm } from '../../SubmissionSummarySection'
+import { DataDetailMissingField } from '../DataDetailMissingField'
+
+type Contact = StateContact | ActuaryContact
+
+function isCertainActuaryContact(contact: Contact): boolean {
```

Comment:
You could make this function a type-check with the return type `: contact is ActuaryContact` and I think you could lose the `as`’s in this file. https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates